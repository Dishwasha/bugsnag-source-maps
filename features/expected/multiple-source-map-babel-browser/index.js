"use strict";

var _a = _interopRequireDefault(require("./lib/a"));

var _b = _interopRequireDefault(require("./lib/b"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

console.log('hi from index');
var b = new _b["default"]();
var a = new _a["default"](b);
a.hello();
//# sourceMappingURL=index.js.map
