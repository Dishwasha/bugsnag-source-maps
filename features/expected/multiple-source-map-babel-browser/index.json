{
    "version": 3,
    "sources": [
        "src/index.js"
    ],
    "names": [
        "console",
        "log",
        "b",
        "B",
        "a",
        "A",
        "hello"
    ],
    "mappings": ";;AAAA;;AACA;;;;AAEAA,OAAO,CAACC,GAAR,CAAY,eAAZ;AAEA,IAAMC,CAAC,GAAG,IAAIC,aAAJ,EAAV;AACA,IAAMC,CAAC,GAAG,IAAIC,aAAJ,CAAMH,CAAN,CAAV;AAEAE,CAAC,CAACE,KAAF",
    "sourcesContent": [
        "import A from './lib/a'\nimport B from './lib/b'\n\nconsole.log('hi from index')\n\nconst b = new B()\nconst a = new A(b)\n\na.hello()\n"
    ],
    "file": "index.js"
}
